cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -277,8 +269,7 @@ spec:
           persistentVolumeClaim:
             claimName: {{ template "cost-analyzer.fullname" . }}
 {{- end }}
-{{- end }}
-{{- if and (.Values.kubecostModel.etlToDisk | default true) .Values.persistentVolume.dbPVEnabled (not (and .Values.kubecostDeployment.statefulSet.enabled .Values.kubecostDeployment.leaderFollower.enabled)) }}
+{{- if .Values.persistentVolume.dbPVEnabled }}
```

Comment:
Value not defined. If this is different from the `enabled` key, then please add with comment.